test/e2e/configs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "bond": 10,
```

Comment:
Could we use `config` in root directory for this?
It lets us set variables based on the environment by creating `environment.js` as defined by `config` module.

For some variables such as txPerBlock, we would want to pull this from the same config that optimist uses too as they will need to be aligned.